src/ResourceManager/Sql/Commands.Sql/packages.config

```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.Azure.KeyVault.Core" version="1.0.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.Sql" version="1.5.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.Network" version="12.0.0-preview" targetFramework="net452" />
```

Comment:
I didn't see where this was used. Please remove.

Reply:
Our tests have a dependency on the Network team's cmdlets. Thus this has been added.

Reply:
This is not the test package config. This is the config for the cmdlets.

Reply:
Thats true. I assumed this was required in the commands project as well. I will try removing it and checking if it works.